gps/verify/lockdiff.go

```diff
@@ -95,26 +95,31 @@ func DiffLocks(l1, l2 gps.Lock) LockDelta {
 			Name: pr1,
 		}
 
-		for i2 := i2next; i2 < len(p2); i2++ {
-			lp2 := p2[i2]
-			pr2 := lp2.Ident().ProjectRoot
-
-			switch strings.Compare(string(pr1), string(pr2)) {
-			case 0: // Found a matching project
-				lpd.LockedProjectPropertiesDelta = DiffLockedProjectProperties(lp1, lp2)
-				i2next = i2 + 1 // Don't visit this project again
-			case +1: // Found a new project
-				diff.ProjectDeltas[pr2] = LockedProjectDelta{
-					Name:         pr2,
-					ProjectAdded: true,
+		// Edge case: If there are no lockedProjects on the RHS, they have most probably been removed from the lock file.
+		if len(p2) == 0 {
+			lpd.ProjectRemoved = true
```

Comment:
Instead of having to outdent everything under an `else` statement, let's just put a `continue` in here.

Reply:
Then I'll have to add  `diff.ProjectDeltas[pr1] = lpd` just before the continue. I'll be duplicating it.